src/js/game/LevelMVC/LevelModel.js

```diff
@@ -599,6 +624,14 @@ module.exports = class LevelModel {
     return this.isPositionEmpty([x, y], entity);
   }
 
+  canMoveBackward(entity = this.player) {
+    const [x, y] = this.getMoveBackwardPosition(entity);
+    if (!this.controller.followingPlayer() && (x > 9 || y > 9)) {
```

Comment:
this condition is a custom check for a specific level, and doesn't need to be replicated in the `moveBackward` logic

Reply:
Fair enough! I was just trying to replicate the whole line of logic to make sure it behaved the same. I'll remove this bit if it's not needed.